pemFioi/quickAlgo/i18n.js

```diff
@@ -30,6 +30,7 @@ var quickAlgoLanguageStrings = {
          display: "Afficher",
       },
       invalidContent: "Contenu invalide",
+      invalidBlock: "Erreur : Le bloc en surbrillance est utilisé mais n'a pas de capteur associé",
```

Comment:
This message is too specific to quick-pi. Maybe something like "Erreur : Le bloc en surbrillance ne fait pas partie des blocs disponibles." ?

Reply:
Changed translation to something less specific